### ISSUE TITLE
Double Messages

### ISSUE THREAD

**Issue by myzinsky** [ChatSecure/ChatSecure-iOS#1158]
Some messages are shown twice. Really annoying while reading conversations...

**Comment by Hund** [comment]
My friend is having the same issue as well with ChatSecure 4.3.9. We're chatting via a Prosody server running version 0.9.12 and Devuan Linux.

I'm not an Ios-user so I have no prior experience how to debug software on that platform. Let me know if you want any more information.

**Comment by myzinsky** [comment]
yeah its really annoying ... please test chat secure with prosody  too :/

**Comment by Hund** [comment]
The last commit made for this project was about 3 months ago, the last closed ticket was about the same time. It doesn't look that bright.

**Comment by chrisballinger** [comment]
Not exactly true, the last commit was about 2 weeks ago, it's just on an
un-merged PR:
https://github.com/ChatSecure/ChatSecure-iOS/pull/1155/commits/f7fdb660d8c576bf4b5e17d4f5fcbe3ac4c79af9

There is also a new 5.0 release available on TestFlight:
https://testflight.apple.com/join/psrh8VB6

**Comment by Hund** [comment]
That's great! I'm looking forward to any future releases.
